hack/build-images.sh

```diff
@@ -120,6 +123,27 @@ image openshift/hello-openshift              examples/hello-openshift
 docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
 docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
 
+# images for integrated services (e.g. logging and metrics)
+clonedir="$(mktemp -d 2>/dev/null || mktemp -d -t clonedir.XXXXXX)"
+pushd $clonedir
+# logging
+git clone --depth 1 --branch ${OS_LOGGING_BRANCH} https://github.com/openshift/origin-aggregated-logging.git 
+image openshift/origin-logging-curator        origin-aggregated-logging/curator
+image openshift/origin-logging-deployment     origin-aggregated-logging/deployment
+image openshift/origin-logging-elasticsearch  origin-aggregated-logging/elasticsearch
+image openshift/origin-logging-fluentd        origin-aggregated-logging/fluentd
+image openshift/origin-logging-kibana         origin-aggregated-logging/kibana
```

Comment:
Yikes.  Ok, that's too long :(

On Thu, Mar 17, 2016 at 4:51 PM, Jeff Cantrill <notifications@github.com>
wrote:

> In hack/build-images.sh
> <https://github.com/openshift/origin/pull/6825#discussion_r56575490>:
>
> > @@ -120,6 +123,27 @@ image openshift/hello-openshift              examples/hello-openshift
> >  docker build --no-cache -t openshift/deployment-example:v1 examples/deployment
> >  docker build --no-cache -t openshift/deployment-example:v2 -f examples/deployment/Dockerfile.v2 examples/deployment
> >
> > +# images for integrated services (e.g. logging and metrics)
> > +clonedir="$(mktemp -d 2>/dev/null || mktemp -d -t clonedir.XXXXXX)"
> > +pushd $clonedir
> > +# logging
> > +git clone --depth 1 --branch ${OS_LOGGING_BRANCH} https://github.com/openshift/origin-aggregated-logging.git
> > +image openshift/origin-logging-curator        origin-aggregated-logging/curator
> > +image openshift/origin-logging-deployment     origin-aggregated-logging/deployment
> > +image openshift/origin-logging-elasticsearch  origin-aggregated-logging/elasticsearch
> > +image openshift/origin-logging-fluentd        origin-aggregated-logging/fluentd
> > +image openshift/origin-logging-kibana         origin-aggregated-logging/kibana
>
> Cleaning my machine of all base images, etc, it took approximately 31
> minutes to build all logging and metrics images. This is on a T530 with 8G
> of RAM
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6825/files/02cb63d1c9842ca49750c396fb560f93804afe5b#r56575490>
>